public/app/features/admin/ServerStats.tsx

```diff
@@ -80,6 +81,12 @@ export const ServerStats = () => {
               { name: 'Organisations', value: stats.orgs },
               { name: 'Users total', value: stats.users },
               { name: 'Active users in last 30 days', value: stats.activeUsers },
+              ...(!!stats.activeDevices && config.featureToggles.anonymousAccess
```

Comment:
Wonder why do we hide this data of no active devices are detected?

Reply:
This is more of a check for values being available `undefined`. It is true though that this check is not needed and should be done when we are displaying the values 👍